File: frontend/src/app/models/entity/ParamRecepDevolBodega.ts
import { ParamDetRecepDevolBodega } from '../../models/entity/ParamDetRecepDevolBodega';

export class ParamRecepDevolBodega {
    constructor(
   
        public hdgcodigo           ?: number,
        public esacodigo           ?: number,
        public cmecodigo           ?: number,
        public servidor            ?: string,
        public usuariodespacha     ?: string,
        public soliid              ?: number,
        public solibodorigen       ?: number,
        public soliboddestino      ?: number,
        public paramdetdevolbodega ?: ParamDetRecepDevolBodega[],
    ) { }   
}

File: backend/Modulos/models/paramRecepDevolBodega.go
package models

// ParamRecepDevolBodega is...
type ParamRecepDevolBodega struct {
	PiHDGCodigo       int                        `json:"hdgcodigo"`
	PiESACodigo       int                        `json:"esacodigo"`
	PiCMECodigo       int                        `json:"cmecodigo"`
	PiServidor        string                     `json:"servidor"`
	PiUsuarioDespacha string                     `json:"usuariodespacha"`
	PiSoliID          int                        `json:"soliid"`
	SOLIBODORIGEN     int                        `json:"solibodorigen"`
	SOLIBODDESTINO    int                        `json:"soliboddestino"`
	Detalle           []ParamDetRecepDevolBodega `json:"paramdetdevolbodega"`
}
